### PR TITLE
fix: update deps

### PR DESCRIPTION
**Pull request by schuettc**
Changes:

- Update dependencies to fix deployment
- Move NestedStacks to Constructs
- Change regex in Receive Digits to collect 8 & 9

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/chime-sdk-stack.ts [hunk 1]
```diff
@@ -27,21 +21,21 @@ export class ChimeSdkStack extends NestedStack {
     this.phoneNumber = phoneNumber.phoneNumber;
 
     const sipMediaApp = new chime.ChimeSipMediaApp(this, 'sipMediaApp', {
-      region: this.region,
-      endpoint: props?.smaLambdaEndpointArn as string,
+      region: Stack.of(this).region,
+      endpoint: props.smaLambdaEndpointArn as string,
```
Comment (scottrfrancis) [inline lib/chime-sdk-stack.ts:25]: is the smaLambdaEndpointArn presence actually guaranteed? What behavior do you want if that prop doesn't exist?

File: src/lambda_function.py [hunk 2]
```diff
@@ -6,256 +6,232 @@
...
+        log_level = "INFO"
 except:
-    log_level = 'INFO'
+    log_level = "INFO"
 logger.setLevel(log_level)
 
 # Load environment variables
-wav_bucket = os.environ['WAVFILE_BUCKET']
+wav_bucket = os.environ["WAVFILE_BUCKET"]
+try:
```
Comment (scottrfrancis) [inline src/lambda_function.py:18]: a simpler construction here would be to use 'get'
dial_number = os.environ.get('DIAL_NUMBER', False)

could do same for WAVFILE_BUCKET... if it's not set line 17 will throw an Exception

File: src/lambda_function.py [hunk 3]
```diff
@@ -6,256 +6,232 @@
...
 
     global log_prefix
-    log_prefix = 'Call-ID:{} {} From:[{}] To:[{}]: '.format(call_id, event_type, from_number, to_number)
-    logger.info('RECV {} {} {}'.format(log_prefix, event_type, 'event received'))
-
-    if event_type == 'NEW_INBOUND_CALL':
-        return new_call_handler(call_id)
+    log_prefix = "Call-ID:{} {} From:[{}] To:[{}]: ".format(call_id, event_type, from_number, to_number)
+    logger.info("RECV {} {} {}".format(log_prefix, event_type, "event received"))
+    if event_type == "NEW_INBOUND_CALL":
```
Comment (scottrfrancis) [inline src/lambda_function.py:36]: can you turn this big if/else chain into a dispatch table?

**Review by scottrfrancis — COMMENTED** [review]
please consider addressing the minor comments before merging.

File: lib/chime-sdk-stack.ts (hunk 1 above)
Comment (schuettc) [reply lib/chime-sdk-stack.ts:25]: Yes, for certain.  This stack will fail entirely without that.  There is no recovery.

File: src/lambda_function.py (hunk 2 above)
Comment (schuettc) [reply src/lambda_function.py]: Changed.

**Review by garnathan — APPROVED** [review]
Approved, but if that double back slash in the regex is a mistake, just do a follow up CR to address.

**Pull request merged**